integration_tests/timbre5/project.clj

```diff
@@ -1,8 +1,8 @@
 (defproject example "1.0.0"
-  :dependencies [[org.clojure/clojure "1.11.1"]
-                 [com.taoensso/timbre "5.2.1"]
-                 [org.slf4j/slf4j-api "1.7.36"]
+            :dependencies [[org.clojure/clojure "1.11.1"]
+                 [com.taoensso/timbre "6.0.4"]
+                 [org.slf4j/slf4j-api "2.0.6"]
                  ; slf4j-timbre will be added here by lein update-in
                  ]
-  :main example.core
-  :aot [example.core])
\ No newline at end of file
+            :main example.core
+            :aot [example.core])
```

Comment:
Any particular reason for changing the indentation?

Reply:
No, auto-format in Cursive and "don't care for whitespace"-setting in git client. No particular reason besides... it's just whitespace. Sorry.